gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `~astropy.time.Time`
+        Astropy time object for the start of the light curve
+    fermi_met_end : `~astropy.time.Time`
+        Astropy time object for the end of the light curve
+    ax : `~matplotlib.axes.Axes` or None
+        Axes
```

Comment:
I think I might be missing a parameter in my sphinx build, because at
docs/_build/html/time/index.html

I only see a little stub
plot_fermi_3fgl_light_curve(name_3fgl, ...)	Plot flux as a function of time for a fermi 3FGL object.